cloudinit/reporting/events.py

```diff
@@ -247,6 +262,7 @@ def __repr__(self):
 
     def __enter__(self):
         self.result = status.SUCCESS
+        self.start_timestamp = time.time()
```

Comment:
`time.monotonic()` may be a better choice for measuring time as it never goes backwards due to system clock changes, etc.